### PR TITLE
SW-2320 update air filter profiles

### PR DESCRIPTION
**Pull request by JFSolution**
update profiles
fix af1
add hose blocked malfunction

### CODE REVIEW

File: octoprint_mrbeam/iobeam/airfilter.py [hunk 1]
```diff
@@ -296,6 +307,11 @@ def pressure_drop_mainfilter(self):
             return mainfilter_pressure_avg - fan_pressure_avg
         return None
 
+    def exhaust_hose_is_blocked(self):
+        if self.model_id in self.AIRFILTER3_MODELS:
+            return self._pressure2 < self.AF3_PRESSURE2_MIN
+        return None
```
Comment (irlaec) [inline octoprint_mrbeam/iobeam/airfilter.py:313]: Would it be more clear to return False?

File: octoprint_mrbeam/iobeam/airfilter.py [hunk 2]
```diff
@@ -337,6 +357,36 @@ def set_temperatures(
         if temperature4 is not None:
             self._temperature4 = temperature4
 
+    def set_connected(self, connected):
```
Comment (irlaec) [inline octoprint_mrbeam/iobeam/airfilter.py]: Having used @property before, maybe we can use @setter here?

Comment (irlaec) [reply octoprint_mrbeam/iobeam/airfilter.py]: Or maybe not because there's quite a bit of logic...

**Review by irlaec — APPROVED** [review]

File: octoprint_mrbeam/iobeam/airfilter.py (hunk 1 above)
Comment (JFSolution) [reply octoprint_mrbeam/iobeam/airfilter.py:313]: I would say no, as we don't know it for sure, it is a unknown state

File: octoprint_mrbeam/iobeam/airfilter.py (hunk 2 above)
Comment (JFSolution) [reply octoprint_mrbeam/iobeam/airfilter.py]: But a setter can have logic, thats why there is a setter and not direct access, right?

**Pull request merged**